wagtail/images/models.py

```diff
@@ -344,6 +344,8 @@ def usage_url(self):
             ],
         ),
         index.FilterField("uploaded_by_user"),
+        index.FilterField("created_at"),
```

Comment:
The lack of this `FilterField` would result in a 500 error when you try to search and order by Newest or Latest. This bug has been around for a while, e.g. try going to http://localhost:8000/admin/images/?&ordering=created_at&q=a in the current build.